src/odbc/include/timestream/odbc/statement.h

```diff
@@ -65,19 +65,21 @@ class IGNITE_IMPORT_EXPORT Statement : public diagnostic::DiagnosableAdapter {
   void BindColumn(uint16_t columnIdx, int16_t targetType, void* targetValue,
                   SqlLen bufferLength, SqlLen* strLengthOrIndicator);
 
+  void Statement::ExtendedFetch(SQLUSMALLINT orientation, SQLLEN offset, SQLULEN* rowCount, SQLUSMALLINT* rowStatusArray);
```

Comment:
Please remove the Statement:: prefix here and in front of InternalExtendedFetch. It breaks non-Windows builds.

Reply:
Removed.